airflow/contrib/hooks/google_discovery_api_hook.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+"""
+This module allows you to connect to the Google Discovery API Service and query it.
+"""
+from googleapiclient.discovery import build
+
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+
+
+class GoogleDiscoveryApiHook(GoogleCloudBaseHook):
```

Comment:
We should also add Client Info as per #5728

Reply:
Client Info is only compatible with google-cloud-* libraries only. Internally Composer use a mockey-patching of HTTP Client to track request.

Reply:
Here is commit: https://github.com/wwlian/composer-airflow/commit/5218e725b821c90a6f98e245f821d40f6616bb25#diff-aeef6fae3fb1fdb6e4663b68518b0d3a

Reply:
Is this still relevant?

Reply:
No. It not relevant.